target-bin/bootstrap-fixup.in

```diff
@@ -16,13 +16,15 @@ elif [ $DISTRIB_NAME = "Debian" ]; then
   # grub-legacy conflicts grub-pc dependencies
   # No grub-legacy on Ubuntu, just on Debian
   # Work around bcron-run conflict due to cron being removed
-  apt-get purge -y grub-legacy bcron-run &> /dev/null
+  # Needed for KVM, but apparently errors out for LXC, so the true ignores the error
+  apt-get purge -y grub-legacy bcron-run &> /dev/null || true
 fi
 echo '127.0.1.1 gitian' >> /etc/hosts
 
 # If LXC
 if grep /lxc/gitian /proc/1/cgroup > /dev/null; then
-  adduser --disabled-password --gecos ubuntu --quiet ubuntu || true
+  adduser --disabled-password --gecos ${DISTRIB_NAME,,} --quiet ${DISTRIB_NAME,,} || true
+  adduser ${DISTRIB_NAME,,} sudo
```

Comment:
I removed the commit that adds the user to the sudo group.

@ageis - I think you should submit an issue about looking into using unprivileged containers. It may be difficult, because the easiest way seems to be to switch to `lxc-create`, but I'm not sure if the templates it ships with are similar enough to what we get from `debootstrap` to use those templates and maintain reproducibility with respect to the KVM builds. I haven't tried getting unprivileged LXC containers working in the context of Gitian though, so I'm not sure. But at least the idea is recorded separately from this PR then.